kernel/libs/page_allocator/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "page_allocator"
```

Comment:
The crate name need to be revised, since we need to publish it and we don't want the user to think it a generic allocator. Name it `aster-page-allocator` instead.

Also, you need to add automatic publishing (`tools/github_workflows/publish_ostd_and_osdk.sh`) and bumping (`tools/bump_version.sh`) for this crate.